boards/xtensa/esp32s3/esp32s3-eye/scripts/Make.defs

```diff
@@ -23,13 +23,26 @@ include $(TOPDIR)/tools/Config.mk
 include $(TOPDIR)/tools/esp32s3/Config.mk
 include $(TOPDIR)/arch/xtensa/src/lx7/Toolchain.defs
 
-ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_out.ld
-ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3.ld
-ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_rom.ld
-ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_peripherals.ld
+# Pick the linker scripts from the board level if they exist, if not
+# pick the common linker scripts.
+
+ARCHSCRIPT += $(BOARD_COMMON_DIR)$(DELIM)scripts$(DELIM)esp32s3_peripherals.ld
+ARCHSCRIPT += $(BOARD_COMMON_DIR)$(DELIM)scripts$(DELIM)esp32s3_rom.ld
+
+ifneq ($(wildcard $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_memory.ld),)
+	ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_memory.ld
```

Comment:
```suggestion
  ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)esp32s3_memory.ld
```
as in other places in code tree

Reply:
Sorry, but which other places?
If we take https://github.com/apache/incubator-nuttx/blob/master/tools/Config.mk as an example, most lines are indented with tabs.

Reply:
From file that you are pointing:
```
# ARCHxxx means the predefined setting(either toolchain, arch, or system specific)

ARCHDEFINES += ${shell $(DEFINE) "$(CC)" __NuttX__}
ifeq ($(CONFIG_NDEBUG),y)
  ARCHDEFINES += ${shell $(DEFINE) "$(CC)" NDEBUG}
endif

# The default C/C++ search path

ARCHINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include}

ifeq ($(CONFIG_LIBCXX),y)
  ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include$(DELIM)libcxx}
else ifeq ($(CONFIG_UCLIBCXX),y)
  ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include$(DELIM)uClibc++}
else
  ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include$(DELIM)cxx}
  ifeq ($(CONFIG_ETL),y)
    ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include$(DELIM)etl}
  endif
endif
ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include}
```
I'm referring to:
```
boards/risc-v/mpfs/icicle/scripts/Make.defs
boards/risc-v/esp32c3/esp32c3-devkit/scripts/Make.defs
boards/arm/samv7/samv71-xult/configs/knsh/Make.defs
```
and some others. Anyway I can unify style later

Reply:
I see that TABs are used for targets, but not for variables

Reply:
Yes, that's exactly my point, NuttX coding standard does not provide any guideline to Makefile formatting and there is simply no consistency as of today.
Still, I've applied your suggestions.